aten/src/ATen/native/native_functions.yaml

```diff
@@ -446,6 +446,7 @@
   dispatch:
     CPU: _s_copy__cpu
     CUDA: _s_copy__cuda
+    QuantizedCPU: _s_copy__quantized
```

Comment:
Just wondering, will we have to support copy between two quantized tensors?

Reply:
I don't think we will support that in this call, we might have other ways to create a QTensor from another QTensor. or provide some other APIs..